influxdb/client.py

```diff
@@ -495,6 +495,32 @@ def drop_database(self, dbname):
         """
         self.query("DROP DATABASE {0}".format(quote_ident(dbname)))
 
+    def get_list_measurements(self):
+        """Get the list of measurements in InfluxDB.
+
+        :returns: all measurements in InfluxDB
+        :rtype: list of dictionaries
+
+        :Example:
+
+        ::
+
+            >> dbs = client.get_list_measurements()
+            >> dbs
+            [{u'name': u'measurements1'},
+             {u'name': u'measurements2'},
+             {u'name': u'measurements3'}]
+        """
+        return list(self.query("SHOW MEASUREMENTS").get_points())
```

Comment:
By checking  InfluxDB 1.2  docs about show measurements.
There was more than one clause needed to support. **on** , **with measurement**, **where**, **limit**, **offset**.
Support all the clauses maybe coming from another PR, and it's better that we could discuss the interface of this feature.